framework/scripts/moosegrep.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Usage:
+#  moosegrep.sh appexecutable objectname inputfile1.i [inputfile2.i ...]
+#
+
+APP=$1
+OBJ=$2
+
+if [ x$3 == 'x' ]
+then 
+  echo "Usage: moosegrep.sh appexecutable objectname inputfile1.i [inputfile2.i ...]"
+  exit 1
+fi
+
+n=0
+for INPUT in $@
+do
+  if [ $n -ge 2 ]
+  then
+    ../../tensor_mechanics-opt -i $INPUT --list-constructed-objects | sed -e '1,/\*\*START OBJECT DATA\*\*/d' -e '/\*\*END OBJECT DATA\*\*/,$d' | grep '^'$OBJ'$' > /dev/null 2>&1
```

Comment:
No, that is an oversight. The script takes an executable. But you are right
accepting a filter parameter would be better.


 hard-coded tensor_mechanics-opt? Do we really need this script? Seems like

most